apidoc/Titanium/Platform/Platform.yml

```diff
@@ -357,10 +357,23 @@ properties:
     platforms: [android, iphone, ipad]
 
   - name: version
-    summary: System's OS version.
+    summary: The operating system's version string.
     type: String
     permission: read-only
 
+  - name: versionMajor
+    summary: The operating system's major version number.
+    type: Number
+    permission: read-only
+    since: "9.2.0"
+
+  - name: versionMinor
+    summary: The operating system's minor version number.
+    description: Will return zero if OS does not have a minor version.
+    type: Number
+    permission: read-only
+    since: "9.2.0"
```

Comment:
``` yml
default: `0`
```

Note that we haven't really tightened the definition for what is a valid "default" value. So it's really just for doc consumption on the site - but ideally we'd be able to more tightly specify this in some way that we could auto-generate tests or something from it.

Reply:
Good idea. I'll do the same for `OS_VERSION_MINOR`.